server.js

```diff
@@ -18,6 +20,6 @@ app.get('/', (req, res) => {
   res.send('hello')
 })
 
-app.listen(PORT, () => {
+app.listen(8080, () => {
```

Comment:
Hey, what if we change this to:

```suggestion
app.listen(PORT, () => {
```